src/antsibull_docs/jinja2/tests.py

```diff
@@ -14,21 +14,22 @@
 from antsibull_core.vendored.collections import is_sequence
 
 
-# if a module is added in a version of Ansible older than this, don't print the version added
-# information in the module documentation because everyone is assumed to be running something newer
-# than this already.
-TOO_OLD_TO_BE_NOTABLE = '0.0.0'
+# The following dictionary maps collection names to cut-off versions. If a version of such a
+# collection is mentioned as when a feature was added that is older than the cut-off version,
+# we do not print the version.
+TOO_OLD_TO_BE_NOTABLE = {
+    'ansible.builtin': '2.7',
+}
 
 test_list: t.Callable[[t.Any], bool] = partial(is_sequence, include_strings=False)
 
 
-def still_relevant(version, cutoff=TOO_OLD_TO_BE_NOTABLE, collection=None):
+def still_relevant(version, collection=None):
     """
     Calculates whether the given version is older than a cutoff value
 
     :arg version: Version to check
-    :arg cutoff: Calculate whether `version` is older than this
-    :returns: True if the `version` is older than `cutoff` otherwise True.
+    :returns: True if the `version` is older than the cutoff version, otherwise True.
```

Comment:
is the end of this supposed to be 'otherwise False' ?

Reply:
the `otherwise True` is correct, but the start should be `False`. Thanks for spotting this!